crates/nu-command/src/generators/seq.rs

```diff
@@ -324,74 +357,110 @@ fn print_seq(
     pad: bool,
     padding: usize,
     span: Span,
+    is_float_sequence: bool,
```

Comment:
Mhh, the data is still coming in as float and may have lost precision. Could we just pass in the `Counter`?